ui/app/components/panel-header.hbs

```diff
@@ -0,0 +1,25 @@
+{{#let
+  (hash
+    title=(or
+      (and (eq @artifact "deployment") "Deployment")
+      (and (eq @artifact "build") "Build")
+      (and (eq @artifact "release") "Release")
+      (and @artifact "n/a")
```

Comment:
We should probably add translations for these. (There’s a linting rule we can enable to help with this)